solutions/devsprint-bruno-ramos-3/DeliveryApp/Screens/RestaurantList/Restaurant.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  RestaurantListRequest.swift
+//  DeliveryApp
+//
+//  Created by Amaryllis Baldrez on 16/02/22.
+//
+
+import Foundation
+import UIKit
+
+struct Restaurant {
+    
+    private let name: String
+    private let category: String
+    private let minTime: Int
+    private let maxTime: Int
```

Comment:
Aqui vamos utilizar o novo modelo de domínio de `DeliveryTime` que voce criou
```suggestion
    private let deliveryTime: DeliveryTime
```